lib/plug/parsers/multipart.ex

```diff
@@ -163,11 +169,12 @@ defmodule Plug.Parsers.MULTIPART do
   end
 
   defp multipart_type(headers) do
-    with {_, disposition} <- List.keyfind(headers, "content-disposition", 0),
+    with {:disp, {_, disposition}} <- {:disp, List.keyfind(headers, "content-disposition", 0)},
          [_, params] <- :binary.split(disposition, ";"),
          %{"name" => name} = params <- Plug.Conn.Utils.params(params) do
       handle_disposition(params, name, headers)
     else
+      {:disp, _} -> {:binary_with_headers, "_parts[]"}
```

Comment:
Let's call it `:part` instead of `:binary_with_headers`, what do you think?

Reply:
Works for me!